dashboard/dashboard/backend/table_state.py

```diff
@@ -62,14 +62,16 @@ def filtered_sorted_items(self) -> List[Item]:
                     ]
                 )
             ]
-
+            self.total_items = len(items)
+        else:
+            self.total_items = len(self.items)
         return items
 
     @rx.var(cache=True)
     def page_number(self) -> int:
         return (self.offset // self.limit) + 1
 
-    @rx.var(cache=True)
+    @rx.var(cache=False)
```

Comment:
does this not work as a cached var anymore? when `total_items` or `limit` changes, it should be recalculated, is that not enough?